lib/kms_monitor/cloudtrail.rb

```diff
@@ -94,27 +176,29 @@ def get_app_record(uuid, timestamp)
       result.item
     end
 
-    def insert_into_db(uuid, timestamp, ctdata, cwdata)
+    def insert_into_db(uuid, timestamp, ctdata, cwdata, _correlated)
```

Comment:
What's the unused correlated argument for?

Reply:
That's a glaring bug. I added a test to the spec, confirmed it failed, then fixed the use of correlated.